conf/app.routes

```diff
@@ -2,7 +2,7 @@
 
 GET         /declaration/:id                        uk.gov.hmrc.cdsreimbursementclaim.controllers.DeclarationController.declaration(id: MRN)
 GET         /declaration/:id/reason-for-security    uk.gov.hmrc.cdsreimbursementclaim.controllers.DeclarationController.declarationWithReasonForSecurity(id: MRN, reasonForSecurity: ReasonForSecurity)
-GET         /declaration/:id/is-duplicate           uk.gov.hmrc.cdsreimbursementclaim.controllers.DuplicateClaimController.isDuplicate(id: MRN, reasonForSecurity: ReasonForSecurity)
+GET         /declaration/:id/claim-exists           uk.gov.hmrc.cdsreimbursementclaim.controllers.ExistingClaimController.claimExists(id: MRN, reasonForSecurity: ReasonForSecurity)
```

Comment:
Remember to make the corresponding change in the frontend bit that calls this endpoint.

Reply:
> Remember to make the corresponding change in the frontend bit that calls this endpoint.

That's a very good point, I'll do that now